cada/commands.py

```diff
@@ -205,15 +206,16 @@ def anon():
     '''Check for candidates to anonymization'''
     header(anon.__doc__)
     filename = 'urls_to_check.csv'
+    name_regex = r'(M\.|Mme\.|Monsieur|Madame|Docteur|Mademoiselle)\s+[^X\s\.]{3}'
```

Comment:
Add `Mr` I guess. It’s not proper French but a lot of people make the mistake.

Reply:
"Mlle" can be added to

Reply:
« Maître » and whatever is its abbreviation(s)

Reply:
`Mme` w/o final point.

Reply:
`Dr` for Docteur

Reply:
http://cada.data.gouv.fr/20120785/ peut être ajouter l'anonymisation des entreprises